hbjs.js

```diff
@@ -329,7 +330,7 @@ function hbjs(instance) {
       * @param {string} script: The buffer script
       */
       setScript: function (script) {
-        var str = createCString(script);
+        var str = createAsciiString(script);
```

Comment:
It probably does not matter in practice, but you can turn the string into a tag yourself and use hb_script_from_iso15924_tag()

Reply:
I may try that and open a followup. I also wonder if there would be a big difference if you can pass an `hb_tag` directly to `setScript` and `setLanguage` (so it accepts `string | number`).